src/api/Models/Stratigraphy.cs

```diff
@@ -41,7 +41,7 @@ public class Stratigraphy
     /// Gets or sets the <see cref="Stratigraphy"/>'s update date.
```

Comment:
REQ: Hier ebenfalls `<inheritdoc />` verwenden. Dito für die anderen Properties in diesem File.